app/javascript/dashboard/components/helpCenter/Header/Header.stories.js

```diff
@@ -0,0 +1,51 @@
+import { action } from '@storybook/addon-actions';
```

Comment:
The story file name should be in the format of `<COMPONENT_NAME>.sories.js`. Here, the file name will be `ArticleHeader.stories.js`